docs/Migration.md

```diff
@@ -3,9 +3,9 @@
 To deploy contracts to a specified network, follow these steps:
 
 1. Set the environment variable `arcjs_network` to the name of the network ("ganache", "kovan", "ropsten", "live"). The default is "ganache".  Truffle will use this setting to find the specified settings in truffle.js in the Arc.js package.  The migration script will also use this setting just to feedback to you which network truffle is using.
-2. The migration script will use `arcjs_gasLimit_deployment`, allowing you to customize the gas amount if needed.
-3. make sure that /migrations/founders.json in the Arc.js package folder has an entry for your test network with the founders to add to the Genesis DAO.
-4. make sure you have your testnet running and listening on the right port.
+2. The migration script will use the gas settings defined the Arc.js file `/gasLimits.js`.  The gas limit when migrating/creating Daos is computed dynamically as a function of the number of founders.
+3. Make sure that `/migrations/founders.json` in the Arc.js package folder has an entry for your test network with the founders to add to the Genesis DAO.
```

Comment:
wait, so to deploy to ganache now we have to add a founders.json file to Arc.js? that seems really weird

Reply:
No, you don't have to add it, it is there just as before, nothing has changed in that respect.  I will add a comment that founders for ganache are already in it.